examples/pytorch/rgcn/entity_classify_mp.py

```diff
@@ -247,20 +279,27 @@ def run(proc_id, n_gpus, args, devices, dataset):
 
     for epoch in range(args.n_epochs):
         model.train()
-        optimizer.zero_grad()
-        if args.sparse_embedding:
-            emb_optimizer.zero_grad()
```

Comment:
is this a bug? we need to invoke `zero_grad` for every batch, right?

Reply:
Basically, we should update gradient every step.
It is fixed in this PR.